bytes/bytes.d.ts

```diff
@@ -0,0 +1,66 @@
+﻿// Type definitions for bytes v2.1.0
+// Project: https://github.com/visionmedia/bytes.js
+// Definitions by: Zhiyuan Wang <https://github.com/danny8002/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare module 'bytes' {
+
+    interface IBytesFormatOptions {
```

Comment:
sorry for this. I am a newer for typescript. thank you for your reminder and i will fix it